helpers/image_helper.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require 'open-uri'
+require 'tempfile'
+require_relative '../config/StaticData'
+
+# Class for working with images after convert service
+class ImageHelper
+  def self.get_image_size(url)
+    Tempfile.create('images-', StaticData::TMP_FOLDER.to_s) do |fo|
+      fo.binmode
```

Comment:
Is binmode really necessary?

Reply:
The binary file mode will stop line-ending conversion.
As a result, data will be more "raw" & you'll avoid breaking parts of the binary file.

Reply:
Yeah, I agree that in some cases binmode is needed (seems if downloading text data)
But compare this:
```ruby
url = 'https://i.imgur.com/vDuB6NZ.png'
Tempfile.create('images-') do |fo|
  fo.binmode
  fo.write(URI.parse(url).open(&:read))
  puts "binmode size: #{File.size(fo.path)}"
end

Tempfile.create('images-') do |fo|
  fo.write(URI.parse(url).open(&:read))
  puts "default mode size: #{File.size(fo.path)}"
end
```

Output will be the same:
```
binmode size: 611141
default mode size: 611141
```

So in your case it's just an extra line, that didn't change anything

Reply:
Yeah, `Encoding::UndefinedConversionError: "\x89" from ASCII-8BIT to UTF-8`
Erorr is not good, ok, seem `binmode` is needed

Reply:
Yes, the file sizes do not change, but my car writes to me:
Encoding::UndefinedConversionError: "\x89" from ASCII-8BIT to UTF-8
And the properties binmod:
-newline conversion disabled -encoding conversion disabled -content is treated as ASCII-8BIT
i think this is important.

Reply:
Yes, the file sizes do not change, but my car writes to me:

`Encoding::UndefinedConversionError: "\x89" from ASCII-8BIT to UTF-8`

And the properties **binmod**:

-newline conversion disabled 
-encoding conversion disabled 
-content is treated as ASCII-8BIT

i think this is important.